docs/index.rst

```diff
@@ -29,6 +29,18 @@ allow for that.
 Overall, skorch aims at being as flexible as PyTorch while
 having a clean interface as sklearn.
 
+If you use Skorch, please use this BibTeX entry:
```

Comment:
```suggestion
If you use skorch, please use this BibTeX entry:
```

Reply:
I’m inclined to use capital letters I think software libraries are proper nouns. But that’s a separate PR.

Reply:
We consistently use lowercase "s" for skorch.